### PR TITLE
Add path tests for timestamp offset

### PR DESCRIPTION
**Pull request by algochoi**
Unit tests for timestamp offset endpoint and the ready endpoint.

Only GetBlockTimestampOffset returns a non-null response, so only that response test was updated.

Tested on the PySDK: https://github.com/algorand/py-algorand-sdk/pull/468

### CODE REVIEW

**Review by Eric-Warehime — APPROVED** [review]

**Comment by winder** [comment]
Please add the new tag to the README. Could you also add the `/ready` endpoint?

**Review by winder — CHANGES_REQUESTED** [review]
Please add the new tag to the README. Could you also add the `/ready` endpoint?

**Comment by algochoi** [comment]
@winder 

Good point about README, I will fix that in the next commit - where should the `/ready` endpoint be added? It looks like version agnostic endpoint tests do not exist right now

**Comment by winder** [comment]
@algochoi good point about the non-v2 test. How about `algodclient_paths.feature`?

File: features/unit/v2algodclient_paths.feature [hunk 1]
```diff
@@ -179,3 +179,19 @@ Feature: Algod REST API v2 Paths
     Examples:
       | path            | method |
       | /v2/ledger/sync | delete |
+
+  @unit.timestamp
+  Scenario Outline: SetBlockTimeStampOffset path
+    When we make a SetBlockTimeStampOffset call against offset <offset>
+    Then expect the path used to be "<path>"
```
Comment (winder) [inline features/unit/v2algodclient_paths.feature:186]: May as well use the new one :shrug:
```suggestion
    Then expect the request to be "get" "<path>"
```

Comment (algochoi) [reply features/unit/v2algodclient_paths.feature]: Updated in : [9ec4a57](https://github.com/algorand/algorand-sdk-testing/pull/276/commits/9ec4a577a4df9c65d140dd8cc8c6ccbdd9f425c4)

**Pull request merged**